arbnode/node.go

```diff
@@ -121,6 +121,9 @@ func (c *Config) Validate() error {
 		c.Feed.Output.Enable = false
 		c.Feed.Input.URL = []string{}
 	}
+	if c.EspressoCaffNode.Enable && (c.Sequencer || c.DelayedSequencer.Enable || c.SeqCoordinator.Enable) {
```

Comment:
should we also check for timeboost here?

Reply:
We wouldn't be able to check for timeboost being enabled here because its part of the sequencer config which doesn't get included in the arbnode config.